program_overview.html

```diff
@@ -109,217 +109,263 @@
-              <li class="">
-              </li>
-            </ol>
-
-            <h3>The Overview of the Conference Program will be as follows.</h3>
-            <div class="spacer"></div>
-
-            <div class="row">
-              <div class="col-sm-8">
-                <table class="table table-striped table-hover table-responsive">
-                  <thead class="thead-default">
-                    <th class="col-sm-4">Day</th>
-                    <th class="col-sm-8">Event</th>
-                  </thead>
-                  <tbody>
-                    <tr>
-                      <th scope="row">Mon. 1</th>
-                      <td class="bg-warning" style="font-size: 20px;">GRADUATE CONSORTIUM + WORKSHOPS</td>
-                    </tr>
-                    <tr>
-                      <th scope="row">Tue. 2</th>
-                      <td rowspan="3" class="bg-info" style="color: white; font-size: 20px;">MAIN CONFERENCE</td>
-                    </tr>
-                    <tr>
-                      <th scope="row">Wed. 3</th>
-                    </tr>
-                    <tr>
-                      <th scope="row">Thur. 4</th>
-                    </tr>
-                  </tbody>
-                </table>
-              </div>
+    </div>
+  </nav>
+
+  <section class="section program">
+    <!-- Page Content -->
+    <div class="container-fluid section-heading">
+
+      <div class="row">
+        <div class="col-sm-1"></div>
+        <div class="col-sm-9">
+          <!-- Page Heading/Breadcrumbs -->
+          <h1 class="mt-4 mb-3">Program at a Glance
+            <small></small>
+          </h1>
+
+          <ol class="breadcrumb">
+            <li class="">
+            </li>
+          </ol>
+
+          <div class="spacer"></div>
+
+          <div class="row">
+            <div class="col-sm-10">
+              <table class="table table-striped table-hover table-responsive">
+                <thead class="thead-default">
+                  <th class="col-sm-3">Pre-conference Events</th>
+                  <th class="col-sm-9" colspan="3">Conference Events</th>
+                </thead>
+                <tbody>
+                  <tr>
+                    <th scope="row">Monday, 1</th>
+                    <th scope="row">Tuesday, 2</th>
+                    <th scope="row">Wednesday, 3</th>
+                    <th scope="row">Thursday, 4</th>
+                  </tr>
+
+                  <tr>
+                    <td class="bg-warning">Workshops</td>
+                    <td class="bg-info text-white">Keynote by Jason Hong</td>
+                    <td class="bg-info text-white">Keynote by Geraldine Fitzpatrick</td>
+                    <td class="bg-info text-white">Keynote by Rodrigo Coutinho</td>
+                  </tr>
+
+                  <tr>
+                    <td class="bg-warning">Graduate Consortium</td>
+                    <td class="bg-info text-white">Main conference</td>
```

Comment:
@jacomecunha I think stating "Main conference" once per line/block is enough, rather than stating it in each cell. Makes it look cleaner